autoware_map_msgs/msg/LaneletMapCellMetaData.msg

```diff
@@ -0,0 +1,7 @@
+# Metadata of lanelet map cell
+
+int32 cell_id
```

Comment:
https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_map_msgs/msg/PointCloudMapCellMetaDataWithID.msg#L3

I would like to unify it since it is defined as a String here.


```suggestion
string cell_id
```

Reply:
updated